app/models/manageiq/providers/kubevirt/infra_manager/vm.rb

```diff
@@ -1,6 +1,6 @@
 class ManageIQ::Providers::Kubevirt::InfraManager::Vm < ManageIQ::Providers::InfraManager::Vm
   include Operations
-
+  include RemoteConsole
   POWER_STATES = {
```

Comment:
Should have a newline after this:

```suggestion

  POWER_STATES = {
```